utils/score.ts

```diff
@@ -15,9 +21,16 @@ function scoreOccurrence(occ: Occurrence): number {
   if (reviewData.reviews.some(r => r.images.length > 0))
     score += 1
 
-  // Punish "always" available dishes
-  if (commonDishes.some(c => occ.dish.nameDe.toLowerCase().includes(c)))
-    score -= 1
+  let highestPenalty = 0
+  for (const commonKey in commonDishPenalty) {
+    if (occ.dish.nameDe.toLowerCase().includes(commonKey)) {
+        const penalty = commonDishPenalty[commonKey]
+        if (penalty > highestPenalty)
+            highestPenalty = penalty
+    }
+  }
+
+  score -= highestPenalty
```

Comment:
Was spricht dagegen einfach die reihenfolge in der die penalties gecheckt werden umzukehren? Also erst höchste penalties testen und dann absteigend. Sollte mmn aufs selbe hinauskommen und weniger code (selber code wie davor). Einfach die einträge in zeile 5-9 nach wert absteigend sortieren.